### ISSUE TITLE
Tk.getMultipleSaveFile does not work on the Mac

### ISSUE THREAD

**Issue by AndyObtiva** [ruby/tk#38]
The following code snippet works for all dialogs except the last one "Tk.getMultipleSaveFile"

```ruby
require 'tk'

p Tk.getOpenFile
p Tk.getMultipleOpenFile
p Tk.getSaveFile
p Tk.getMultipleSaveFile
```

I get this error message:

```ruby
/Users/andy/.rvm/gems/ruby-3.0.2@glimmer-dsl-tk/gems/tk-0.4.0/lib/tk.rb:2938:in `_invoke': bad option "-multiple": must be -defaultextension, -filetypes, -initialdir, -initialfile, -message, -parent, -title, -typevariable, -command, or -confirmoverwrite (RuntimeError)
	from /Users/andy/.rvm/gems/ruby-3.0.2@glimmer-dsl-tk/gems/tk-0.4.0/lib/tk.rb:2938:in `_invoke'
	from /Users/andy/.rvm/gems/ruby-3.0.2@glimmer-dsl-tk/gems/tk-0.4.0/lib/tk.rb:1983:in `_ip_invoke_core'
	from /Users/andy/.rvm/gems/ruby-3.0.2@glimmer-dsl-tk/gems/tk-0.4.0/lib/tk.rb:2016:in `_tk_call_core'
	from /Users/andy/.rvm/gems/ruby-3.0.2@glimmer-dsl-tk/gems/tk-0.4.0/lib/tk.rb:2039:in `tk_call'
	from /Users/andy/.rvm/gems/ruby-3.0.2@glimmer-dsl-tk/gems/tk-0.4.0/lib/tk.rb:1939:in `getMultipleSaveFile'
	from tmp/test_get_multiple_save_file.rb:6:in `<main>'
```

It seems like ActiveTcl-8.6.9.8609.2-macosx10.9-x86_64-93b04018.pkg has dropped support for `Tk.getMultipleSaveFile`

I'll try to confirm this behavior/issue on other platforms once I get a chance.

If it is true, I could perhaps contribute a Pull Request to remove that method (or provide a better error message on the Mac if it is a Mac-only issue)

**Comment by AndyObtiva** [comment]
The oldest offered [ActiveTcl at ActiveState.com](https://www.activestate.com/products/tcl/) is ActiveTcl 8.5. I just tried it on the Mac. It doesn't support passing the multi option to getSaveFile either (i.e. does not support getMultipleSaveFile)

**Comment by jeremyevans** [comment]
Looks like Tk itself documents `tk_getOpenFile` and `tk_getSaveFile` together (https://www.tcl-lang.org/man/tcl8.5/TkCmd/getOpenFile.htm).  The `-multiple` option is listed, but it says it only applies to the Open dialog. The only documentation that references `getMultipleSaveFile` is in the `README.macosx-aqua` file, and the code itself is untouched since it was added in 2005. I suppose it is possible that `getMultipleSaveFile` never worked.  In any case, unless another committer objects, I'm OK removing it.

**Comment by jeremyevans** [comment]
Closed by #39.
